app/adminPage.html

```diff
@@ -49,17 +49,41 @@
     <title>Admin Page</title>
 </head>
 <body>
-    <!-- Modal -->
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous" defer></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous" defer></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous" defer></script>
+
     <div class="overlay close-view-results"></div>
-    <div id="modal">
-        <h3 id="modal-title"></h3>
-        <button class="close-edit-user primary-button">Close</button>
-        <div id="modal-content"></div>
+
+    <div class="modal bg-transparent" id="modal" tabindex="-1" role="dialog" data-toggle="modal">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-body">
+                    <div id="modal-content"></div>
+                </div>
+                <div class="modal-footer">
+                    <button class="primary-button" id="editSubmit">Submit changes</button>
+                    <button type="button" class="primary-button close-edit-user">Close</button>
+                </div>
+            </div>
+        </div>
     </div>
-    <div id="view-results-modal">
-        <button class="close-view-results primary-button">Close</button>
-        <div id="view-results-modal-content"></div>
+
+    <div class="modal bg-transparent" id="view-results-modal" tabindex="-1" role="dialog">
```

Comment:
what is the purpose of the `bg-transparent` class? it doesn't seem to be making any difference - if it's not needed remove it

Reply:
remove from css and other modal too - it seems like the class was specifically made for this (and doesn't do anything)

Reply:
.bg-transparent has been removed. It seemed so important at the time!